projects/eval-adxl355-pmdz/src/platform/maxim/main.c

```diff
@@ -100,6 +98,8 @@ int main()
 		return ret;
 
 	ret = iio_trigger_example_main();
+#else
+#error IIO_TRIGGER_EXAMPLE is currently supported only on max32655 and max78000 targets.
```

Comment:
Isn't this check already done in the `parameters.h` file?

Reply:
yes, it is already done, I removed it from the main file now.